src/WelcomePlain.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { FHIRWrap } from './FHIRClientWrapper';
+import PatientSearch from './PatientSearch';
+
+const defaultFhirServer = 'https://syntheticmass.mitre.org/fhir';
+
+class Welcome extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { fhirServer: defaultFhirServer, decedentName: '' };
+    this.handlePatientSelect = this.handlePatientSelect.bind(this);
+    this.updateFhirServer = this.updateFhirServer.bind(this);
+  }
+
+  handlePatientSelect(patient) {
+    FHIRWrap.loadResources(this.state.fhirServer, patient.id).then(([conditions, medications, procedures, observations]) => {
+      this.props.setResources(conditions, medications, procedures, observations);
+    });
+    this.props.setPatient(patient);
+    this.props.gotoStep('Pronouncing');
+  }
+
+  updateFhirServer(fhirServer) {
+    this.setState({ fhirServer });
```

Comment:
Could this inadvertently overwrite (or even remove) `decedentName` in this.state? Should it be: `this.setState({ fhirServer: fhirServer });`? I'm actually thinking maybe `decedentName` isn't required here anymore.

Reply:
`this.setState({ fhirServer });` and `this.setState({ fhirServer: fhirServer });` are actually equivalent JavaScript, and neither impact `decedentName` (state is updated via a shallow merge).

Reply:
Ahh I see, it's shorthand from ES6 (if variable name is the same as the key, just include the variable). Not really a fan, since it's not explicit, but not worth changing.